home/models.py

```diff
@@ -193,6 +199,12 @@ class WhatsappBlock(blocks.StructBlock):
         "media cannot exceed 1024 characters.",
         validators=(MaxLengthValidator(4096),),
     )
+    example_values = blocks.ListBlock(
+        ExampleValuesBlock(),
```

Comment:
Do we need the children here to be `StructBlock`s? Could we not just do a `blocks.ListBlock(blocks.CharBlock(....))`?

Reply:
🤦 Thanks, fixed